custom-steps/tinyxml/post-build.ps1

```diff
@@ -0,0 +1,14 @@
+param (
+    [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
+)
+
+Import-Module "$PSScriptRoot/../../ps-modules/Util"
+
+$IsOnMacOS = Get-IsOnMacOS
+
+if(-not $IsOnMacOS) {
+    Write-Host "The step is only required for MacOS.  Skipping step..."
+    exit
+}
+
+Write-Host "Running post-build script..."
```

Comment:
This script doesn't do anything. Can we just remove it completely? same with the pre-build

Reply:
Tinyxml is only even in here because I was using it as a test project to run the process end-to-end, since it is such a small library.  Leave it and put in a comment to that effect in these scripts, or take it out completely?  What do you think?

Reply:
I think just take it out. There are enough other examples to follow if necessary.

Reply:
I still think it is still useful to have this one for now, since it just runs so fast.  I've removed the extraneous Write-Host messages from the other build scripts and put a comment at the top of the pre-and-post build scripts for tinyxml to indicate they are only there for testing purposes.